en/docs/learn/api-security/oauth2/access-token-types/jwt-tokens.md

```diff
@@ -43,13 +43,19 @@ The following are the mandatory attributes that are required for a JWT access to
 
 Follow the instructions below to work with JWT Authentication in WSO2 API Manager.
 
-### Step 1 - Access the Developer Portal
+### Step 1 - Enable a jwt in toml file
+ 
+ [apim.jwt]
+    
+  enable = true
```

Comment:
In this document, we discuss about generating a JWT access token instead of an opaque token. So you don't need to enable apim.jwt. This configuration is for enabling backend jwt which is sent to the backend service.
We don't need to have this change.